client/signup/config/steps-pure.js

```diff
@@ -39,8 +38,6 @@ export function generateSteps( {
 	isDomainFulfilled = noop,
 	maybeRemoveStepForUserlessCheckout = noop,
 	createSiteAndAddDIFMToCart = noop,
-	excludeStepIfEmailVerified = noop,
-	excludeStepIfProfileComplete = noop,
```

Comment:
Can we also remove these completely? They don't seem to be used anymore.

Reply:
Good point. I've cleaned them up in 58d8d1b6509. They look quite p2-flow specific.